.gitignore

```diff
@@ -10,15 +10,17 @@ desktop.ini
 .DS_Store
 ._*
 
-# Travis
-/build/phpdoc-*/
-/build/phpdoc-*.git/
-
 # Composer
 /composer.lock
 /composer.phar
 /vendor
 
+# phpDocumentor
+/_build/phpdoc/
```

Comment:
If I get you well, you changed the build target directory to _build in order to avoir collision with the directory containing the phpdoc shell scripts.

Could we have rather moved the scripts to a "bin" directory and leave the build target dir untouched ? Hard to tell what is the difference between build and _build if you're not familiar with the project.

Just my two cents.